apis/v1alpha1/gateway_types.go

```diff
@@ -281,6 +283,20 @@ type Listener struct {
 	// are compatible with the application protocol specified in
 	// the Protocol field.
 	//
+	// Multiple Routes may match a request to this Listener. If that happens,
+	// precedence MUST be given in order of the following criteria:
```

Comment:
Nit: add a newline here so this formats as a markdown unordered list.

Reply:
Updated, thanks!